Simplify data mapper with unlocalized and localized dimension content (#205)
sulu/SuluContentBundle: UPGRADE.md

# Upgrade

## 0.7.0

### ContentMapperInterface changed

> The `ContentMapperInterface` has changed.

Reviewer: ```suggestion
The `ContentMapperInterface` was changed as a preparation for refactoring the `DimensionContentCollection`:
```

**Before**

```php
    public function map(
        array $data,
        DimensionContentCollectionInterface $dimensionContentCollection
    ): void;
```

**After**

```php
    public function map(
        DimensionContentCollectionInterface $dimensionContentCollection,
        array $dimensionAttributes,
        array $data
    ): void;
```

### DataMapperInterface changed

> The `DataMapperInterface` has changed to make the mapper easier to set localized and unlocalized data.

Reviewer: ```suggestion
The `DataMapperInterface` was changed to make it easier to set localized and unlocalized data:
```
